iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/writer/WriterBuilder.java

```diff
@@ -95,7 +96,9 @@ public HiveIcebergWriter build() {
     int currentSpecId = table.spec().specId();
     int partitionId = attemptID.getTaskID().getId();
     int taskId = attemptID.getId();
-    String operationId = queryId + "-" + attemptID.getJobID();
+    Map<String, List<HiveIcebergWriter>> writers = WriterRegistry.writers(attemptID);
+    int writerCount = writers == null ? 0 : writers.size();
+    String operationId = queryId + "-" + attemptID.getJobID() + "-" + writerCount;
```

Comment:
What about adding a random UUID adding to this, or using an AtomicInteger?

Reply:
Added AtomicInteger instead of getting the number of writers.